lighthouse-core/audits/works-offline.js

```diff
@@ -41,14 +41,16 @@ class WorksOffline extends Audit {
    */
   static audit(artifacts) {
     let debugString;
+    const passed = artifacts.Offline === 200;
     if (!URL.equalWithExcludedFragments(artifacts.URL.initialUrl, artifacts.URL.finalUrl)) {
-      debugString = 'WARNING: You may be failing this check because your test URL ' +
-          `(${artifacts.URL.initialUrl}) was redirected to "${artifacts.URL.finalUrl}". ` +
-          'Try testing the second URL directly.';
+      const explanation = passed ? 'Your test URL' :
```

Comment:
do we want to show this at all if they passed? If they passed, it means that the `initialUrl` successfully redirected to `finalUrl` and `finalUrl` successfully loaded, so it seems like everything is good in that case?